src/main/scala/is/hail/HailContext.scala

```diff
@@ -223,6 +224,11 @@ class HailContext private(val sc: SparkContext,
       }
   }
 
+  def getTemporaryFile(nChar: Int = 10,
+  prefix: Option[String] = None, suffix: Option[String] = None): String = {
```

Comment:
Can you reformat this code? I think it would look nicer if prefix was on the first line. I also don't think you need the `{` for a one line function.